fluid_onnx/ops.py

```diff
@@ -333,37 +389,22 @@ def dropout_op(operator, block):
         ratio=attrs['dropout_prob'])
 
     ## Fluid and ONNX use different dropout formula
-    # ONNX 1.0.1 doesn't support Scale op
-    if __onnx_ver__ == '1.0.1':
-        scale_val = [outputs['Out'][0] + '@scale']
-        constant_node = make_node(
-            'Constant',
-            inputs=[],
-            outputs=scale_val,
-            value=make_tensor(
-                name=scale_val[0],
-                dims=(),
-                data_type=TensorProto.FLOAT,
-                vals=[1.0 - attrs['dropout_prob']]))
-        mul_node = make_node(
+    # onnx version >= 1.4.1 supports ai.onnx ver 9 which has no `broadcast` attr for Mul
+    scale_mul_attrs = {'broadcast': 1} if __onnx_ver__ < '1.4.1' else {}
+    scale_const_node = global_const(np.array(1.0 - attrs['dropout_prob'], dtype=np.float32), block)
+    return _to_node_tuple([
+        dropout_node,
+        scale_const_node.node,
+        make_node(
             'Mul',
-            inputs=scale_input + scale_val,
-            outputs=outputs['Out'],
-            broadcast=1)
-        nodes = (dropout_node, constant_node, mul_node)
-    else:
-        scale_node = make_node(
```

Comment:
According to [the ONNX code line](https://github.com/onnx/onnx/blob/d7595f348c84d7202b057a2541699c643b5b190c/onnx/checker.cc#L511) and [list of latest ONNX operators](https://github.com/onnx/onnx/blob/master/docs/Operators.md), the `Scale` op is deprecated and that many tools like `onnx-tensorflow` has not implemented. So I would suggest remove this block for the sake of standardization.